packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -215,6 +205,9 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
           updatableFieldInput.defaultValue !== undefined
             ? updatableFieldInput.defaultValue
             : existingFieldMetadata.defaultValue,
+        ...this.prepareCustomFieldMetadataOptions(
+          fieldMetadataInput.options ?? existingFieldMetadata.options,
```

Comment:
why are we defaulting to the existing options, can't we just avoid passing it in case we don't have it in the input?

Reply:
Indeed, this implies validating twice a data that has already been validated ( as it has already been inserted ) 
But I need the options here to validate the `defaultValue`, I did not wanted to introduce a new abstraction such as existingFieldMetadataOptions, so I merged it there 🤔

***This means that we will face runtime issue while this is deployed if we do not handle legacy*** You're right this is not good, I will add the just above described abstraction to handle this gracefully and not block legacy entries